tests/theta_test.py

```diff
@@ -0,0 +1,105 @@
+"""Tests for theta.py"""
+
+import pytest
+import os
+import tempfile
+import git
+import random
+import string
+
+from git_theta import theta, git_utils
+from tests import testing_utils
```

Comment:
I have never understood the best way to have importable code in tests. If this is the best way to do it just leave it, otherwise can you add a TODO (or an issue) to update to the best way (whatever pytest recommends lol)